julia_src/ghpghx_results.jl

```diff
@@ -0,0 +1,104 @@
+
+Base.@kwdef mutable struct ResultsStruct
```

Comment:
Do the results need to be mutable? It is safer to keep any data struct immutable.

Reply:
Yes! This struct gets updated at every iteration (sub-hourly typically) of the ghpghx simulation period by the GHX Fortran subroutine. For each sizing iteration in the sizing solver, it gets reset with the zero values assigned using the kwargs functionality of this struct.